website_baidu_map/README.rst

```diff
@@ -0,0 +1,39 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+website_baidu_map
+=================
+
+This module extends the functionality of website to support baidu map in the contact us page.
```

Comment:
Maybe my code reading is improper but AFAIR, the map is link to event, not "contact us" page.
Can you recheck?